picard/tagger.py

```diff
@@ -1036,7 +1036,9 @@ def longversion():
 
 def process_picard_args():
     parser = argparse.ArgumentParser(
-        epilog="If one of the filenames begins with a hyphen, use -- to separate the options from the filenames."
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        epilog="""If one of the filenames begins with a hyphen, use -- to separate the options from the filenames.
+If -c/-P/-s are not provided, your arguments will be ignored by an existing instance."""
```

Comment:
Technically, even if these are provided the arguments will be ignored by an existing instance, won't they?  I mean they will trigger a new stand-alone instance.  The arguments that will be ignored by an existing instance are `-d`, `-M` and `-N`.

Reply:
Hm, right, how about replacing *an existing instance* with *the target instance*? Or maybe:

*If a new instance won't be spawned, the other aguments will be ignored*

Reply:
That works, although it could be interpreted that the files / directories for processing will also be ignored.  Perhaps something like, "*If a new instance won't be spawned, all arguments other than files and directories for processing will be ignored.*" or "*If a new instance won't be spawned, all user interface related arguments will be ignored.*"?

Reply:
Good point, please check the newest commit

Reply:
The wording looks fine to me.  One minor issue...  Should there be a space between the bullet indicator (-) and the text of the bullet?  For example:

```
If a new instance will not be spawned:
  - files/directories will be passed to the existing instance,
  - the other arguments will be ignored.
```

rather than:

```
If a new instance will not be spawned:
  -files/directories will be passed to the existing instance,
  -the other arguments will be ignored.
```

Reply:
Whatever, as long as they are consistent, imo

Reply:
With space I'd say